cli/util/send.go

```diff
@@ -37,6 +40,28 @@ func sendTx(ctx *cli.Context) error {
 	if err != nil {
 		return cli.NewExitError(fmt.Errorf("failed to submit transaction to RPC node: %w", err), 1)
 	}
+	if ctx.Bool("await") {
+		acc, w, err := options.GetAccFromContext(ctx)
+		if err != nil {
+			return cli.NewExitError(fmt.Errorf("failed to get account from context to sign the conflicting transaction: %w", err), 1)
+		}
+		defer w.Close()
+
+		signers, err := cmdargs.GetSignersAccounts(acc, w, nil, transaction.CalledByEntry)
+		if err != nil {
+			return cli.NewExitError(fmt.Errorf("invalid signers: %w", err), 1)
+		}
+		a, actorErr := actor.New(c, signers)
+		if actorErr != nil {
+			c.Close()
+			return cli.NewExitError(fmt.Errorf("failed to create Actor: %w", actorErr), 1)
+		}
+		_, err = a.Wait(tx.Hash(), tx.ValidUntilBlock, nil)
+		if err != nil {
```

Comment:
The case is that we don't need actor to perform awaiting here. And we don't have any accounts or wallets. What we need is: in a separate commit make `func newWaiter(ra RPCActor, v *result.Version) Waiter` exported and replace `RPCActor` argument type with `any`. Make an appropriate comment about acceptable values. After that in a subsequent commit you can directly pass RPC client to this waiter constructor and expect it to work properly.